apps/auth/service.py

```diff
@@ -130,9 +130,11 @@ def delete(self, lookup):
         if error_message:
             raise SuperdeskApiError.forbiddenError(message=error_message)
 
-        # Delete all the sessions
+        # Delete all the sessions except current session
         for session in sessions:
-            get_resource_service("auth").delete_action({config.ID_FIELD: str(session[config.ID_FIELD])})
+            current_session_id = auth.get_auth().get("_id")
```

Comment:
not that it would be a big deal, but this should be done before the loop, it won't change inside so makes me wonder why it's there

Reply:
sorry my bad, I got your point.